src/fu-util.c

```diff
@@ -2678,7 +2700,12 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Reinstall current firmware on the device."),
 		     fu_util_reinstall);
-
+	fu_util_cmd_array_add (cmd_array,
+		     "get-hsi",
```

Comment:
What do you think about `get-security` instead of `get-hsi` for this?

Reply:
What about just `security` instead?

Reply:
That seems fine to me. Don't forget to add it to bash completion too.

Reply:
Fixed, thanks.